### ISSUE TITLE
Doesn't support optional quotes tag attribute syntax?

### ISSUE THREAD

**Issue by jmk2142** [dekimasoon/language-riot-tag#2]
`<div class="{ show:true, popular:isPopular }">This syntax highlighting works</div>`
`<div class={ show:true, popular:isPopular }>This syntax highlighting breaks</div>`

![screen shot 2016-02-27 at 4 12 43 pm](https://cloud.githubusercontent.com/assets/13051415/13375434/088f42bc-dd6d-11e5-850b-ffee79fe0115.png)

But so far just having this package available has been very helpful. Thanks!

**Comment by dekimasoon** [comment]
Hi @jmk2142 , excuse my premature issue closing.

I read [your comment](https://github.com/jumilla/atom-riot/issues/6) on the other Riot syntax package. It was really helpful and I update my package 0.2.0 which supports `optional quotes tag attribute syntax`.

Could you please check if it is OK?

**Comment by jmk2142** [comment]
Hey. Glad it could help. To be honest with you, I'm new to the custom grammar creation myself so I'm not sure what is best practice to naming conventions of different patterns. But I love Riot and I think this is a useful package.

Your code:

```
    name: 'string.quoted.other.riot'
```

I use:

```
    'name': 'meta.string.expression.riot'
    'contentName': 'source.js'
```

No idea what meta means in this context but string makes sense since the original form is "{ }" and { } is just an optional way of typing it. Also convenient as there is already a default style set for it. I include the class `expression` instead of `other` because that properly explains the semantics of this grammar with what the tag will wrap. `{ model }` is a Riot expression. The last part riot is self explanatory.

What I would also suggest (and maybe there is a better way is to also include the property `'contentName': 'source.js'`. This takes whatever is inside the beginCapture and endCapture and wraps that in it's own tag within Atom. Thus, Atom will be aware that the contents of the expression are javascript... I think. And that is the Riot way - expressions are pure JS.

One more thing - on yours, the id attribute seems a little wonky. I see the `id="something"` and `"something"` part not highlighting green like the other attribute quoted values. In your grammar, the `"something"` is coming up as just plain text where it probably should be wrapped in something like

```
<span class="string quoted double html">
    <span class="punctuation definition string begin html">"</span>
        something
    <span class="punctuation definition string end html">"</span>
</span>
```

**Comment by dekimasoon** [comment]
Thank you for your comment :+1:
I'm also new to this TextMate's grammar and no idea what is the best practice at all too.
Sorry for your inconvenience. I'll find some time in this weekend to fix the `id` syntax issue and consider what you've suggested.

**Comment by dekimasoon** [comment]
I've merged your PR and released v0.2.1 ;)
Thank you from the bottom of my heart.

**Comment by jmk2142** [comment]
NP. This is a helpful addition to Atom for Riot fans. The other repo doesn't seem to be very active right now - author hasn't fixed the bugs nor responded to issues. So for now let's keep improving this one. ;-)
